### ISSUE TITLE
文本分类Demo因读入Dataset训练时未对batch做padding引起Blocking queue is killed because the data reader raises an exception

### ISSUE THREAD

**Issue by vForce825** [PaddlePaddle/PaddleNLP#46]
最小复现代码：
import paddle
import paddlehub as hub
model = hub.Module(name='roberta-wwm-ext-large', task='seq-cls', num_classes=2)
train_dataset = hub.datasets.ChnSentiCorp(
    tokenizer=model.get_tokenizer(), max_seq_len=128, mode='train')
dev_dataset = hub.datasets.ChnSentiCorp(
    tokenizer=model.get_tokenizer(), max_seq_len=128, mode='dev')
optimizer = paddle.optimizer.Adam(learning_rate=5e-5, parameters=model.parameters())
trainer = hub.Trainer(model, optimizer, checkpoint_dir='test_ernie_text_cls')
trainer.train(train_dataset, epochs=3, batch_size=32, eval_dataset=dev_dataset)

原因排查：
观察发现报错为：
File "/home/daniel/anaconda3/lib/python3.7/site-packages/paddle/fluid/dataloader/dataloader_iter.py", line 90, in default_collate_fn
    tmp = np.stack(slot, axis=0)
排查一个batch内的数据，发现每个样本的shape不相同，所以引起np.stack报错
